css/style.css

```diff
@@ -0,0 +1,1248 @@
+}
+.icon-arrow-down:before {
+  content: "\e610";
+}
+.icon-google-plus:before {
+  content: "\e603";
+}
+.icon-facebook:before {
+  content: "\e604";
+}
+.icon-twitter:before {
+  content: "\e605";
+}
+.icon-dribbble:before {
+  content: "\e606";
+}
+.icon-linkedin2:before {
+  content: "\e607";
+}
+.icon-menu:before {
+  content: "\e60f";
+}
+.icon-drop:before {
+  content: "\e60e";
+}
+.icon-plus:before {
+  content: "\e608";
+}
+.icon-chevron-left:before {
+  content: "\e609";
+}
+.icon-chevron-right:before {
+  content: "\e60a";
+}
+.icon-brush:before {
+  content: "\e60b";
+}
+.icon-phone:before {
+  content: "\e60c";
+}
+.icon-pin:before {
+  content: "\e60d";
+}
+
+
+/*
+SECTION HEADERS
+*/
+
+.team-section-header,
+.client-section-header,
+.section-header,
+.sub-section-header,
+.portfolio-section-header,
+.contact-section-header {
+  width: 100%;
+  text-decoration: underline;
+}
+
+.section-header,
+.sub-section-header {
+  max-width: 90%;
+  margin: 0 auto;
+  padding: .75em 0;
+  text-align: center;
+}
+
+@media screen and (min-width: 50em) {
+  .white-line {
+    background: url("/img/horizontal-white-line.svg") no-repeat left,
+                url("/img/horizontal-white-line.svg") no-repeat right;
+  }
+
+  .section-header {
+    background-size: 35%;
+  }
+
+  .sub-section-header {
+    -webkit-flex-basis: 100%;
+    flex-basis: 100%;
+    margin: 0 6em;
+    padding: 1.333em 2em;
+    background-size: 28%;
+  }
+
+  .section-arrows-white:before,
+  .section-arrows-orange:before,
+  .section-arrows-black:before {
+    position: absolute;
+    content: "";
+    display: block;
+    width: 20%;
+    height: 43px;
+    top: 0;
+    left: 40%;
+  }
+
+  .section-arrows-orange:before {
+    background: url("/img/triangle-orange.svg") no-repeat;
+  }
+
+  .section-arrows-black:before {
+    background: url("/img/triangle-black.svg") no-repeat;
+  }
+
+  .section-arrows-white:before {
+    background: url("/img/triangle-white.svg") no-repeat;
+  }
+}
+
+/*
+MENU
+*/
+
+.menu {
+  position: fixed;
+  width: 100%;
+  padding-bottom: 0.5em;
+  border-bottom: 2px solid #d4d4d4;
+  background-color: rgba(255, 255, 255, 0.95);
+  z-index: 1000;
+}
+
+.menu a {
+  display: inline-block;
+  width: 30%;
+  color: #7c8387;
+  font-weight: bold;
+  padding-top: 0.5em;
+}
+
+@media screen and (min-width: 50em) {
+  .menu {
+    position: static;
+    padding: 0.8em calc(20% - 3em);
+  }
+
+  .menu a {
+    padding-top: 0.1em;
+    width: 16%;
+  }
+}
+
+/*
+HEADER & LOGO
+*/
+
+.header-content {
+  padding: 3em 0;
+  background-size: cover;
+  background-image: url(/img/header-background.jpg);
+}
+
+.logo {
+  opacity: 1;
+  height: 4em;
+  width: 4em;
+  margin: 1em auto 1em;
+  padding-top: 1em;
+  padding-bottom: 1em;
+  background: #fd6632;
+}
+
+.header-logo-text strong {
+  font-weight: normal;
+}
+
+.curve-underline {
+  padding-bottom: .1em;
+  background-size: cover;
+  background: url("/img/orange-underline.svg") no-repeat;
```

Comment:
While this works, it gets clipped when you resize the text, so the dropping end of the line gets lost (along with some of the personality).